bodhi/server/models.py

```diff
@@ -839,6 +839,7 @@ class Release(Base):
     pending_testing_tag = Column(UnicodeText, nullable=False)
     pending_stable_tag = Column(UnicodeText, nullable=False)
     override_tag = Column(UnicodeText, nullable=False)
+    template = Column(UnicodeText, default=u'default_template', nullable=True)
```

Comment:
Since the `default_template.tpl` is the same as `fedora_errata_template.tpl`, we could use directly `fedora_errata_template.tpl` that way we don't need to have the `default_template.tpl` file